client/js/store/procedure/SplitStatementStore.js

```diff
@@ -219,6 +219,9 @@ const SplitStatementStore = {
           }
           const initialData = data.data.attributes.segmentDraftList.data
           const segments = initialData.attributes.segments
+            // Filter out segments with less than 10 characters as those may lead the frontend to crash
```

Comment:
```suggestion
            // Filter out segments with less than 10 characters as those may lead the frontend to crash
            // (because often that are closing or opening tags)
```